css/properties.json

```diff
@@ -6397,7 +6397,7 @@
     "groups": [
       "CSS Masking"
     ],
-    "initial": "no-repeat",
+    "initial": "repeat",
```

Comment:
this is already landing in https://github.com/mdn/data/commit/b36469e253ba551ba2a5a9738b814522309e5f6a